src/condor_q.V6/queue.cpp

```diff
@@ -566,8 +566,7 @@ int main (int argc, const char **argv)
 
 			scheddAddr = strdup(schedd.addr());
 			if( (tmp = schedd.name()) ) {
-				scheddName = strdup(tmp);
-				Q.addSchedd(scheddName);
+				Q.addSchedd(tmp);
```

Comment:
scheddName is used below. You have to set it. Either free() it after the usage or change it to a std::string.